backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -1373,3 +1373,23 @@ def test_literature_detail_page_do_not_have_acquisition_source_for_non_curator(
         response = client.get(f"/literature/{control_number}", headers=headers)
     assert response.status_code == 200
     assert "acquisition_source" not in response.json["metadata"]
+
+
+def test_primary_arxiv_category_is_added_when_arxiv_eprints_present(inspire_app):
+    data = {
+        "titles": [{"title": "test test test"}],
+        "arxiv_eprints": [
+            {"categories": ["nucl-th", "astro-ph"], "value": "nucl-th/9310030"},
+            {"categories": ["astro-ph"], "value": "astro-ph/9310030"},
```

Comment:
This test doesn't allow you to see the difference between taking all categories from the first element in `arxiv_eprints` and all first elements of `categories`.